lib/ClaimVerifier.ts

```diff
@@ -63,8 +67,16 @@ export class ClaimVerifier {
       );
       return null;
     }
-    if (!(await this.checkRoleStatus(credential.payload))) {
-      return null;
+    const roleStatus = await this.checkRoleStatus(credential.payload);
+    if (!roleStatus.verified) {
+      return {
+        name: role.roleName,
+        namespace: credentialClaimData.claimType,
+        status:
+          roleStatus.error === CredentialRevoked
+            ? RoleCredentialStatus.REVOKED
+            : RoleCredentialStatus.EXPIRED,
```

Comment:
The code as-is is a bit brittle because if a new error is added but this code isn't changed, then `RoleCredentialStatus.EXPIRED` would be returned incorrectly. What about this change below?
```suggestion
      const errorMessageMap = {
        CredentialExpired: RoleCredentialStatus.EXPIRED,
        CredentialRevoked: RoleCredentialStatus.REVOKED
      }
      return {
        name: role.roleName,
        namespace: credentialClaimData.claimType,
        status: errorMessageMap[roleStatus.error] ?? RoleCredentialStatus.UNKNOWN
```

Reply:
@jrhender I agree with you, but then if a new `error` is added then we would need to update error mapping and add error to `RoleCredentialStatus` as well to make it meaningful. So the code would be needed to update anyhow.